src/core/ProductTile/ProductLabel.tsx

```diff
@@ -51,18 +59,18 @@ const ProductLabel = ({
           "ui-text-p2 font-bold",
           {
             "text-neutral-000 dark:text-neutral-1300":
-              selected !== false && !unavailable,
+              selected === true && !unavailable,
```

Comment:
is it same as?

```
{
  "text-neutral-000 dark:text-neutral-1300": selected && !unavailable,
},
{
  "text-neutral-1000 dark:text-neutral-300": !selected && !unavailable,
},
```

Reply:
Not quite. We have cases for where `selected` is `true`, `false` and `undefined`, so in this case we need a way of distinguishing `false` and `undefined`, which `!selected` wouldn't (since it casts `undefined` as `false`